macros/special-notes.cfg

```diff
@@ -1,11 +1,20 @@
 #textdomain wesnoth-Flight_Freedom
 
 # wmlindent: start ignoring
+#define SPECIAL_NOTES_PRECISION
+    _ "This unit has precision attacks, which always have a very high chance of hitting an opponent." #enddef
+
+#define SPECIAL_NOTES_NOCOUNTERATTACK
+    _ "Some of this unit's attacks are not subject to enemy counter-attack." #enddef
+
+#define SPECIAL_NOTES_ALLAROUND
+    _ "Some of this unit's attacks can strike all adjacent targets." #enddef
+
 #define SPECIAL_NOTES_DEFEND_ONLY
-    _ "Some of this unit’s attacks are not available except on defense." #enddef
+    _ "Some of this unit's attacks are not available except on defense." #enddef
```

Comment:
why replace the fancy apostrophe with the plain kind?

Reply:
For consistency's sake with most of the other apostrophes that were plain, but agree that fancy apostrophe looks better.

Reply:
ok, I guess that's fine.